src/ceph_mon.cc

```diff
@@ -140,7 +140,8 @@ int check_mon_data_empty()
   errno = 0;
   while ((de = ::readdir(dir))) {
     if (string(".") != de->d_name &&
-	string("..") != de->d_name) {
+	string("..") != de->d_name &&
+	string("kv_backend") != de->d_name) {
```

Comment:
kv_backend is not being created correctly monitor db is created and looking and the PendingReleaseNotes I'm not sure if thats the intention.

With the described semantics I was unable to find a way to create monitor with rocksdb. 
ceph-mon.mkfs calls store.create_and_open (here https://github.com/ceph/ceph/blob/master/src/ceph_mon.cc#L409) and that expects the kv_backend file to already exist (see https://github.com/ceph/ceph/blob/master/src/mon/MonitorDBStore.h#L642). If I create the kv_backend file then check_mon_data_emtpy (here https://github.com/ceph/ceph/blob/master/src/ceph_mon.cc#L129) fails.